backend/jamrequestmodule/views.py

```diff
@@ -140,6 +140,40 @@ def retrieve(self, request, *args, **kwargs):
         })
 
 
+@api_view(('GET',))
+def getUserGenres(request, user_id):
+    user_genres = UserGenre.objects.filter(userid=user_id).all()
+    genres = [user_genre.genreid for user_genre in user_genres]
+    ser_genres = MusicGenreSerializer(genres, many=True)
+    return Response({
+        'genres': ser_genres.data
+    })
+
```

Comment:
I couldn't figure out how to do queries by user_id for these tables using classes (didn't see how to pass the user_id) so I did them as functions.  Is this a problem to mix class-based models with function-based?

Reply:
No sir, this is just fine. I think for our purposes it is fine to have both formats.

For reference, I believe you could do the class based method like this:

```py
class UserReviewByUserId(RetrieveAPIView):
    lookup_field = 'userid'
    queryset = UserReview.objects.all()
    serializer_class = UserReviewSerializer
```